include/ck/tensor_operation/gpu/device/masking_specialization.hpp

```diff
@@ -47,13 +50,39 @@ struct MaskOutUpperTrianglePredicate
         return operator()(m + m_tile - 1, n);
     }
 };
+struct MaskUpperTriangleFromBottomRightPredicate
+{
+    MaskUpperTriangleFromBottomRightPredicate() : offset_(0) {}
+    __host__ __device__ void SetOffset(const index_t offset) { offset_ = offset; }
```

Comment:
Rename `offset`  and `offset_` to `diagonal_offset` and `diagonal_offset_`  respectively,   and add some comments to show the user how to calculate the `diagonal_offset`

Reply:
Also rename `SetOffset` to `SetDiagonalOffset`

Reply:
done